src/dom/dom.js

```diff
@@ -2221,29 +2240,40 @@ p5.prototype.createCapture = function(...args) {
   let useAudio = true;
   let constraints;
   let callback;
+  let flipped = false;
+
   for (const arg of args) {
     if (arg === p5.prototype.VIDEO) useAudio = false;
     else if (arg === p5.prototype.AUDIO) useVideo = false;
-    else if (typeof arg === 'object') constraints = arg;
-    else if (typeof arg === 'function') callback = arg;
+    else if (typeof arg === 'object') {
+      if (arg.flipped !== undefined) {
+        flipped = arg.flipped;
+        delete arg.flipped;
+      }
+      constraints = Object.assign({}, constraints, arg);
+    }
+    else if (typeof arg === 'function') {
+      callback = arg;
+    }
   }
-  if (!constraints) constraints = { video: useVideo, audio: useAudio };
 
+  const videoConstraints = { video: useVideo, audio: useAudio };
+  constraints = Object.assign({}, videoConstraints, constraints);
   const domElement = document.createElement('video');
   // required to work in iOS 11 & up:
   domElement.setAttribute('playsinline', '');
-
   navigator.mediaDevices.getUserMedia(constraints).then(function (stream) {
     try {
       if ('srcObject' in domElement) {
         domElement.srcObject = stream;
       } else {
         domElement.src = window.URL.createObjectURL(stream);
       }
-    } catch (err) {
+    }
+    catch(err) {
       domElement.src = stream;
     }
-  }, console.log);
+  }, console.err);
```

Comment:
Oh I just noticed, I don't think `console.err` is defined, I think this should be `console.error`:
```suggestion
  }, console.error);
```